autogpt_platform/backend/backend/copilot/baseline/service.py

```
@@ -977,9 +977,19 @@ async def stream_chat_completion_baseline(
                     msg["content"] = prefixed
                     break
-            user_message_for_transcript = prefixed
-        else:
-            logger.warning("[Baseline] No user message found for context injection")
+            if prefixed is not None:
+                # Persist the prefixed content so subsequent turns and --resume
+                # retain the user context.
+                for idx, session_msg in enumerate(session.messages):
+                    if session_msg.role == "user":
+                        session_msg.content = prefixed
+                        await update_message_content_by_sequence(
```

Ruff (0.15.9)

[error] Undefined name `update_message_content_by_sequence`

(F821)